site/content/docs/main/file-system-backup.md

```diff
@@ -138,7 +138,7 @@ To mount the correct hostpath to pods volumes, run the node-agent pod in `privil
 
 If node-agent is not running in a privileged mode, it will not be able to access pods volumes within the mounted 
 hostpath directory because of the default enforced SELinux mode configured in the host system level. You can 
-[create a custom SCC](https://docs.openshift.com/container-platform/3.11/admin_guide/manage_scc.html) to relax the 
+[create a custom SCC](https://docs.openshift.com/container-platform/4.16/authentication/managing-security-context-constraints.html) to relax the 
```

Comment:
How about:
https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html

Reply:
The path might break in the future. I've seen couple docs where bumping version result in 404.

Reply:
Done, updated.